### PR TITLE
PG-1025 Doc update for PG 17 and release notes

### PR DESCRIPTION
**Pull request by nastena1606**

### CODE REVIEW

**Review by janwieremjewicz — COMMENTED** [review]
LGTM though I cannot find Everest mentioned. Can we put the Everest somewhere?
Also, is there a chance to mention the Percona Squad for PG already in the release notes? https://squad.percona.com/pg

**Comment by nastena1606** [comment]
@janwieremjewicz please advise in what context do you want Everest mentioned? 
Also we can add Squad to the Services banner at the bottom of the page. Will it work?

File: docs/release-notes-v17.0.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+This release of Percona Distribution for PostgreSQL is based on **Percona Server for PostgreSQL 17** - a binary compatible drop in replacement of [PostgreSQL 17 :octicons-link-external-16:](https://www.postgresql.org/docs/17/release-17.html). Percona Server for PostgreSQL 17 extends the Storage Manager API to hook in custom storage managers and introduce the encryption of indexes (experimental feature) as part of the Transparent Data Encryption (TDE) solution. 
+
+Both Percona Server for PostgreSQL 17 and PostgreSQL Community 17 function identically enabling you to migrate from one to another. However, index-level encryption is available only with Percona Server for PostgreSQL. So if you encrypted indexes, you cannot switch to the upstream PostgreSQL without losing this data.
```
Comment (janwieremjewicz) [inline docs/release-notes-v17.0.md]: I would use a softer statement, pointing to the need to migrate the migration to the method supported on Community edition before migrating to Community PostgreSQL

File: docs/release-notes-v17.0.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+## Release Highlights
+
+Percona Distribution for PostgreSQL 17 features a lot of new functionalities and enhancements to performance, replication, monitoring, developer experience and more. Among them are the following:
```
Comment (janwieremjewicz) [inline docs/release-notes-v17.0.md]: I think we should give what's due to the upstream here pointing that these come with PostgreSQL 17 comes packed with these improvements and that Percona Distribution gets all the same benefits plus the extension of the API from Percona

**Review by janwieremjewicz — COMMENTED** [review]
Other than 2 comments LGTM

File: docs/index.md [hunk 3]
```diff
@@ -14,12 +14,13 @@
 - No guesswork on finding the right version of a component – they all undergo thorough testing to ensure compatibility
 - Freely available reference architectures for solutions like high-availability, backups and disaster recovery 
 - Spatial data handling support via PostGIS
+- Transparent Data Encryption via `pg_tde`.
 - Monitoring of the database health, performance and infrastructure usage via open source [Percona Management and Monitoring :octicons-link-external-16:](https://www.percona.com/doc/percona-monitoring-and-management/2.x/index.html) with PostgreSQL-specific dashboards
 - Run PostgreSQL on Kubernetes using open source [Percona Operator for PostgreSQL :octicons-link-external-16:](https://docs.percona.com/percona-operator-for-postgresql/2.0/index.html). It not only automates deployment and management of PostgreSQL clusters on Kubernetes, but also includes enterprise-ready features for high-availability, backup and restore, replication, logging, and more
```
Comment (janwieremjewicz) [inline docs/index.md:18]: Could we add an Everest bullet here the same way we have one for the Operator?

**Review by janwieremjewicz — APPROVED** [review]

File: docs/index.md (hunk 3 above)
Comment (nastena1606) [reply docs/index.md:18]: Added

File: docs/release-notes-v17.0.md (hunk 2 above)
Comment (nastena1606) [reply docs/release-notes-v17.0.md]: Updated

File: docs/release-notes-v17.0.md (hunk 1 above)
Comment (nastena1606) [reply docs/release-notes-v17.0.md]: Reworded

File: docs/postgresql-server.md [hunk 4]
```diff
@@ -5,10 +5,8 @@ Percona Server for PostgreSQL is a binary-compatible, drop-in replacement for Po
 * Storage Manager (SMGR) API Exposure: Allows PostgreSQL extensions to integrate custom storage managers. This change was inspired by the [patchset](https://www.postgresql.org/message-id/flat/CAJ7c6TOqqrzjYsU6LgDkcJ0yVgzdkx2juJjgAjzP2jPOpZ1qUA%40mail.gmail.com#8e68cfc57fcac14c8e24b00b41e61baf) introduced to the community.
 * WAL Read/Write API Exposure to hook into WAL read and write functions.
 
-These modifications provide index-level and WAL encryption of indexes via the `pg_tde` extension. Note that these encryption features are experimental and under active development.
+These modifications are the preparations to provide index-level and WAL encryption of indexes via the [`pg_tde` :octicons-link-external-16:](https://percona-lab.github.io/pg_tde/main/) extension. These encryption features are still under active development and are planned for future releases.
```
Comment (janwieremjewicz) [inline docs/postgresql-server.md]: Can we rephrase to say that these changes are not impacting on existing use cases unless a specific pg_tde extension build is used? 

These changes can potentially be only used by opt-in to a tech preview of such a build.

Phrasing it in the current fashion suggests that the patch is under active development while the reality is that the extension is

**Review by janwieremjewicz — APPROVED** [review]
Other than the comment LGTM

**Review by Naeem-Akhter — APPROVED** [review]
Looks good to me!

File: docs/release-notes-v17.0.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
+# Percona Distribution for PostgreSQL 17.0.1 ({{date.17_0}})
+
+[Installation](installing.md){.md-button}
+[Upgrade](major-upgrade.md){.md-button}
+ 
+
+We are pleased to announce the launch of Percona Distribution for PostgreSQL 17.0.1 - a solution with the collection of tools from PostgreSQL community that are tested to work together and serve to assist you in deploying and managing PostgreSQL. The aim of Percona Distribution for PostgreSQL is to address the operational issues like High-Availability, Disaster Recovery, Security, Observability, Spatial data handling, Performance and Scalability and others that enterprises are facing.
+
+This release of Percona Distribution for PostgreSQL is based on **Percona Server for PostgreSQL 17.0.1** - a binary compatible drop in replacement of [PostgreSQL Community 17.0 :octicons-link-external-16:](https://www.postgresql.org/docs/17/release-17.html).
```
Comment (janwieremjewicz) [inline docs/release-notes-v17.0.md]: can we add "open source"? as in "a binary compatible, open source, drop in ...

**Review by janwieremjewicz — APPROVED** [review]

**Pull request merged**